components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -211,11 +266,16 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
     this.saveSuccessful = false;
   }
 
-  public saveNameChange(): void {
+  public updateTeam(): void {
     this.saveSuccessful = false;
     this.saving = true;
-    const name: string = this.updateNameForm.controls.name.value.trim();
-    this.store.dispatch(new UpdateTeam({ ...this.team, name }));
+    const newName: string = this.updateNameForm.controls.name.value.trim();
+    this.store.dispatch(new UpdateTeam({
+        id: this.team.id,
+        name: newName,
+        guid: this.team.guid, // to be deprecated after GA
```

Comment:
What does this mean? This is not part of an API or anything, it's our frontend TS code wrapping some backend thing. If we want the code here to look different, we should just do it 😉

Reply:
I think I added this comment a while back. We can drop it, it's just to say we can drop that field, and all the other "versionize" logic, once we've force upgraded everyone to v2

Reply:
💭It would be a nicer experience to drop it if it was centralized to one place, instead of sprinkled all over it 😉

Reply:
Agreed, don't think that's in the scope of this PR though. A lot we could clean up about v1 vs v2 teams in the frontend code.

Reply:
I think the point is more that we don't need that comment here. This should work:
```
    const name: string = this.updateNameForm.controls.name.value.trim();
    const projects: string[] = Object.values(this.projects).filter(p => p.checked).map(p => p.id);
    this.store.dispatch(new UpdateTeam({ ...this.team, name, projects }));
```